### DISCUSSION TITLE
'body' sends content twice?

### DISCUSSION THREAD

**Discussion by courdegard** [sabuhish/fastapi-mail#85] [Q&A]
Feel free to delete this, it started working

Hi, great package. I am probably making a simple mistake here but when my emails send, the body text is repeated. The message looks like 

    message = MessageSchema(
        subject=subject,
        recipients=[email_to],
        body=bodyText,
        subtype='html',
    )

When "bodyText"=="Example", the mail says ExampleExample. When bodyText=="Example.html" where Example.html looks like ``<p> example </p> ``, the mail says 

``<p> example </p> ``

example

When I get rid of the subtype argument I don't see the formatted html, which makes sense. I'm not using an html template but I don't see why that would make a difference in the case of just passing in a string. I can't imagine this is an actual issue with the code so I'm wondering what I have done wrong. Thanks.

**Comment by bhutanict** [comment]
How did you solve it?

html_message = """
<p>Hi this test mail, thanks for using Fastapi-mail</p> 
"""
message = MessageSchema(
            subject="Test",
            recipients=email.dict().get("email"),  # List of recipients, as many as you can pass 
            body=html_message,
            subtype='html'
            )

In the email, I see two content
<p>Hi this test mail, thanks for using Fastapi-mail</p> 

Hi this test mail, thanks for using Fastapi-mail

**Reply by spacetardigradum** [reply]
You have to use, inside messageSchema the attribute html instead of body: 

    html=html_message,
    subtype="html"

That way you get to send the full html with style, links etc with no double content. 
You can find it lower in the docs (the main example is still not working).

**Comment by sabuhish** [comment]
Correct, instead of body please use `html`.
